operators/api/v1alpha1/common.go

```diff
@@ -49,8 +25,13 @@ type GenericRef struct {
 	Namespace string `json:"namespace,omitempty"`
 }
 
-// WorkspaceLabelPrefix is the prefix of a label assigned to a tenant indicating it is subscribed to a workspace.
-const WorkspaceLabelPrefix = "crownlabs.polito.it/workspace-"
+// NameCreated contains information about the status of a resource created in
+// the cluster (e.g. a namespace). Specifically, it contains the name of the
+// resource and a flag indicating whether the creation succeeded.
+type NameCreated struct {
+	// The name of the considered resource.
+	Name string `json:"name,omitempty"`
 
-// TnOperatorFinalizerName is the name of the finalizer corresponding to the tenant operator.
-const TnOperatorFinalizerName = "crownlabs.polito.it/tenant-operator"
+	// Whether the creation succeeded or not.
+	Created bool `json:"created"`
+}
```

Comment:
Is this used?

Reply:
I have no idea, I'll check

Reply:
It seems to me only v1alpha2 is used, but I would double-check.